### DISCUSSION TITLE
problem client from mibbit get block from connectban

### DISCUSSION THREAD

**Discussion by mark4638** [inspircd/inspircd#2017] [Support]
Hello, I am using the INSPIRCD version and I wanted to ask if there is an option in the next version to please add some protection parameters for clients that connect via mibbit because when you activate the module they were all blocked from the m_connectban module
thanks for help

**Comment by sadiepowell** [comment]
You need to configure [the cgiirc module](https://docs.inspircd.org/3/modules/cgiirc/#cgihost) for Mibbit users. That way the connectban module will act on their actual IP address instead of the gateway address. You can also [exempt users from connectban via their connect class](https://docs.inspircd.org/3/modules/connectban/#connect).

You might also want to consider using a hosted client like [Kiwi IRC](https://kiwiirc.com/nextclient/) instead as Mibbit is incredibly outdated and doesn't really support many modern IRC features.
